state-chain/cf-integration-tests/src/swapping.rs

```diff
@@ -573,12 +573,16 @@ fn can_process_ccm_via_direct_deposit() {
 
 		witness_call(RuntimeCall::EthereumIngressEgress(
 			pallet_cf_ingress_egress::Call::contract_ccm_swap_request {
-				source_asset: Asset::Flip,
+				source_asset: cf_chains::assets::eth::Asset::Flip,
```

Comment:
nit: This is bit verbose (same in the btc code in smart_contract.rs). A nice convention used elsewhere in the code is to `use cf_chains::assets::eth::Asset as EthAsset`.